ui/app/AppLayouts/Profile/Sections/StoreToKeychainSelectionModal.qml

```diff
@@ -0,0 +1,77 @@
+import QtQuick 2.13
+import QtQuick.Controls 2.13
+import "../../../../imports"
+import "../../../../shared"
+import "../../../../shared/status"
+
+ModalPopup {
+    id: popup
+
+    title: qsTr("Store pass to Keychain")
+
+    onClosed: {
+        destroy()
+    }
+
+    Column {
+        anchors.top: parent.top
+        anchors.bottom: parent.bottom
+        anchors.left: parent.left
+        anchors.right: parent.right
+        anchors.rightMargin: Style.current.padding
+        anchors.leftMargin: Style.current.padding
+
+        spacing: 0
+
+        ButtonGroup {
+            id: openLinksWithGroup
+        }
+
+        StatusRadioButtonRow {
```

Comment:
@0kok0 no it's not missing, we don't refer to it anywhere, if you wonder why do we have id set for other two `StatusRadioButtonRow` (`notNowBtn` and `neverBtn`) that's only because we set their states from this one (take a look into `onRadioCheckedChanged` slot).